src/components/TypesOfRecipeCards/RecipeCard.jsx

```diff
@@ -0,0 +1,73 @@
+import { PushPin } from "@phosphor-icons/react";
+
+const RecipeCard = ({ pinned, recipe }) => {
+  const white = "#d4d4d4";
+  const size = 18;
+  return (
+    <div>
+      {recipe && (
+        <div className="h-auto w-[165px] object-contain">
+          <div className="relative">
+            <div className="flex">
+              <div className="absolute m-4 rounded-3xl px-4 py-1 text-white backdrop-blur-md">
+                {/* TODO - figure out what makes a recipe trending */}
+                <p className="text-xxs ">Trending</p>
+              </div>
+              {/* PIN for when user pins a recipe */}
+              {pinned && (
+                <div className="absolute right-4 top-4">
+                  <PushPin size={size} color={white} weight="fill" />
+                </div>
+              )}
+            </div>
+
+            <img
+              src={recipe.picturePath}
+              className="h-full w-full rounded-3xl object-cover"
+            />
+            <div className="absolute bottom-2 ml-4 flex flex-col text-white">
+              <div className="flex items-baseline	">
```

Comment:
```suggestion
              <div className="flex items-baseline">
```
Remove whitespaces from this file